src/site/topic16.rst

```diff
@@ -2,8 +2,18 @@
 Topic #16 --- Exceptions
 ************************
 
-Runtime Errors
-==============
+Errors
+======
+
+* In Java, there are two classes of throwable objects
+    * A *throwable* object is one that can be thrown with a ``throw`` statement
+
+* Errors are one of the two throwable classes
+* Errors are typically for unrecoverable situations
+    * They're generally abnormal situations
```

Comment:
Generally **very** abnormal, like the JVM was installed incorrectly or the hard drive fell out. You can also inject errors by screwing with bytecode at runtime (reflection). `AssertionError` also falls into this category, which is easily the most common `Error` one will ever see.